src/index.html

```diff
@@ -1,13 +1,389 @@
+                      alt="place6"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="place__discription">
+              <a href="#" class="place__link"> Audio sunglasses </a>
+            </div>
+          </div>
+
+          <div class="grid grid--disable">
+            <div
+              class="grid__item grid__item--table-2-6 grid__item--desktop-5-9"
+            >
+              <a href="#" class="button">View all</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section argument" id="argument">
+        <div class="container">
+          <div class="grid grid--disable">
+            <div
+              class="grid__item grid__item--table-1-4 grid__item--desktop-1-5"
+            >
+              <h2 class="argument__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+
+            <div
+              class="grid__item grid__item--table-4-7 grid__item--desktop-6-10"
+            >
+              <p class="argument__conditions">
+                Free 2-day shipping and returns
+                <br>
+                90-day risk-free trial
+                <br>
+                World class customer service
+                <br>
+                My Bose account management
+              </p>
+              <p class="argument__about">
+                A great product is more than what’s in the box. It’s a promise
+                of premium performance, world-class support, and everything you
+                expect from a trusted brand. It’s just one of many reasons why
+                you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+
+          <div class="grid grid--disable">
+            <div
+              class="
+              grid__item
+              grid__item--table-1-4
+              grid__item--desktop-1-5"
+            >
+              <form action="get" class="contacts__form">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="contacts__input"
+                >
+
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="contacts__input"
+                >
+
+                <textarea
+                  name="Message"
+                  cols="30"
+                  rows="10"
+                  placeholder="Message"
+                  class="contacts__input contacts__input--textaria"
+                >
+                </textarea>
```

Comment:
```suggestion
                <textarea
                  name="Message"
                  cols="30"
                  rows="10"
                  placeholder="Message"
                  class="contacts__input contacts__input--textaria"
                ></textarea>
```

It should fix extra spaces in textarea field

Reply:
таке питання а як я мав сам написати це?
You need to add some js to clear your form after button click

Reply:
такого ще не було 🤔

Reply:
It seems like you've done it :+1: :fire: